src/bin/cql-stress-cassandra-stress/settings/command/help.rs

```diff
@@ -13,24 +13,24 @@ pub fn print_help() {
     Options::print_generic_help();
 }
 
-fn print_command_help(command_str: &str) -> Result<CommandParams> {
+fn print_command_help(command_str: &str) -> Result<Option<CommandParams>> {
     match Command::parse(command_str) {
         Ok(cmd) => {
             cmd.print_help();
-            Ok(CommandParams::Special)
+            Ok(None)
```

Comment:
> This commit introduces very subtle changes in the settings module.
> It helps the users of the settings access the configuration in a more
> convenient way.

I'd appreciate if the first part of the commit message were less vague. A commit should concentrate on doing one thing, and "very subtle changes in the settings module" is very permissive with regard to what can happen there.

It looks like the idea is to replace `CommandParams` with `Option<CommandParams>` and get rid of `CommandParams::Special`, right? I think you should mention it. The justification that it's more convenient (idiomatic?) is already good.

Besides, a more specific description slightly helps me because I don't have to spend as much time trying to _guess_ the exact idea behind the commit, instead I can concentrate on _verifying_ whether your code aligns with your intentions.